bittensor_cli/src/commands/wallets.py

```diff
@@ -532,11 +541,13 @@ async def overview(
 
         # Pull neuron info for all keys.
         neurons: dict[str, list[NeuronInfoLite]] = {}
+        print_verbose("Fetching subnet netuids", status)
```

Comment:
Same here

Reply:
And yes, working good
![image](https://github.com/user-attachments/assets/9409f566-41ef-4c4f-abd0-bf6fc2058e49)